server/services/socket.js

```diff
@@ -65,27 +49,23 @@ exports.emitNewRound = (status, roundNum, currentPlayer) => {
 
 // Emit the new status of the game
 exports.emitStatus = status => {
-    // console.log("emitStatus", status);
     io.emit("status", {
         status: status
     });
 }
 
 // Tell all the players what word was played
 exports.emitWord = word => {
-    // console.log("emitWord");
     io.emit("played word", word);
 }
 
 // When everyone has played their cards, send them all to the players
 exports.emitPlayedCards = cards => {
-    // console.log("emitPlayedCards", cards);
     io.emit("played cards", cards);
 }
 
 // Ask the current player for a word and a card
 const promptCurrentPlayer = currentPlayer => {
-    // console.log("promptCurrentPlayer", currentPlayer.username);
     for (let socket of sockets) {
         if (socket.handshake.session.user === currentPlayer.username) {
             socket.emit("play word and card");
```

Comment:
Not from this PR but just a thought. This could look a bit nicer if you used `find`. Something like:
`sockets.find(socket => socket.handshake.session.user === currentPlayer.username).emit("play word and card");`
You can split that up too if you think it would be clearer.